### ISSUE TITLE
Drag&drop of fields in composite page edit is not intuitive

### ISSUE THREAD

**Issue by tiberiuichim** [plone/volto#1067]
The problem is that the drop destination doesn't have the area around the handler, so if I just drag a handle over another handle, up or down, it won't "catch", it's only possible if I drag the handle over the content of another block. I've created a video here: https://www.youtube.com/watch?v=AZQkuoVkHIY to show what I mean

**Comment by tisto** [comment]
@tiberiuichim I fully agree with you. This also annoys me a lot as a user. We need somebody to look into this and making a proposal and a possible implementation how to fix this. :)

**Comment by sneridagh** [comment]
@tiberiuichim @tisto yeah, for that we need to refactor a bit the UI, making the block wrapper in the edit UI a bit wider, so the handler can be inside that area, then the drop will be done as expected. As a workaround you should go inner when you drop.

Highlight the area as you drag would be great. I would migrate to use `react-beautiful-dnd` there.
